core/bin/external_node/src/config/mod.rs

```diff
@@ -38,13 +37,16 @@ pub(crate) struct RemoteENConfig {
     pub state_transition_proxy_addr: Option<Address>,
     pub transparent_proxy_admin_addr: Option<Address>,
     pub diamond_proxy_addr: Address,
-    pub l1_erc20_bridge_proxy_addr: Address,
-    pub l2_erc20_bridge_addr: Address,
-    pub l1_weth_bridge_proxy_addr: Option<Address>,
+    pub l1_shared_bridge_proxy_addr: Address,
+    pub l2_shared_bridge_addr: Address,
+    pub l1_erc20_bridge_proxy_addr: Option<Address>,
```

Comment:
I can see that `l1_shared_bridge_proxy_addr` is now mandatory. 

Typically, we will give out the new external node _before_ upgrade so that when it happens everyone is ready. What values will be used there?

Reply:
En will crashed once it asked the new server, please make it optional